azurerm/config.go

```diff
@@ -293,6 +295,12 @@ func (c *Config) getArmClient() (*ArmClient, error) {
 	img.Sender = autorest.CreateSender(withRequestLogging())
 	client.imageClient = img
 
+	ehtc := eventgrid.NewTopicsClientWithBaseURI(endpoint, c.SubscriptionID)
```

Comment:
Nit: What does the `ehtc` mean here in this context? Shouldn't it be `egt` (Event Grid Topics) or something like that?